src/components/Note/Note.scss

```diff
@@ -140,6 +134,20 @@ $note__footer-gap: 42px;
   }
 }
 
+.note__text {
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  display: -webkit-box;
+}
+
+.note__marker-edited {
+  color: rgb(163, 166, 170);
+  font-size: 13px;
```

Comment:
try using the constant colors and spacing found in `constans/style.scss` wherever possible (in this case `$color-middle-gray` and `$spacing--sm`)

Reply:
Also use a lighter gray for dark mode (try looking which color we used for other components?)